cinch/roles/jenkins_master/tasks/plugins.yml

```diff
@@ -56,7 +56,7 @@
 
 - name: perform plugin install
   jenkins_plugin:
-    url: "{{ _jenkins_url }}"
+    url: "http://localhost:8080"
```

Comment:
Because of problems with SSL verification when using self-signed SSL certiifcate. This task would fail as the SSL cert could not be verified.